zipkin/src/main/scala/org/http4s/zipkin/algebras/CollectorOp.scala

```diff
@@ -0,0 +1,14 @@
+package org.http4s.zipkin.algebras
+
+import org.http4s.zipkin.models.ZipkinInfo
+
+import scalaz.Free
+
+// We might consider just making this a trait with an abstract `send` method.
+sealed trait CollectorOp[A]
+
+object CollectorOp {
+  final case class Send(zipkinInfo: ZipkinInfo) extends CollectorOp[Unit]
+  def send(zipkinInfo: ZipkinInfo): Collector[Unit] =
+    Free.liftFC(Send(zipkinInfo))
```

Comment:
Does this still work across scalaz-7.1 and scalaz-7.2?  I think `Free` changed quite a bit.  We cross build for both, off the same branch, now.